libs/gundeck-types/src/Gundeck/Types/Push/V2.hs

```diff
@@ -343,21 +333,19 @@ data PushToken = PushToken
     , _tokenApp       :: !AppName
     , _token          :: !Token
     , _tokenClient    :: !ClientId
-    , _tokenFallback  :: !(Maybe Transport)
     } deriving (Eq, Ord, Show)
 
 makeLenses ''PushToken
 
 pushToken :: Transport -> AppName -> Token -> ClientId -> PushToken
-pushToken tp an tk cl = PushToken tp an tk cl Nothing
+pushToken tp an tk cl = PushToken tp an tk cl
 
 instance ToJSON PushToken where
     toJSON p = object
         $ "transport" .= _tokenTransport p
         # "app"       .= _tokenApp p
         # "token"     .= _token p
         # "client"    .= _tokenClient p
-        # "fallback"  .= _tokenFallback p
```

Comment:
Given that this is used by clients, I'm actually wondering if this will not break things on the client side; giving it a 2nd thought, we might need to keep it around for a while. Can you please check with clients @fisx